src/styles/Onboarding.css

```diff
@@ -114,7 +114,7 @@
 
 .getstarted-button {
```

Comment:
```suggestion
.get-started-button {
```

Make sure to change it in the other parts of the code as well